source/pymongo.txt

```diff
@@ -15,15 +15,19 @@ PyMongo
 Introduction
 ------------
 
-**PyMongo** is the official MongoDB Python driver for MongoDB. We
-recommend you use this driver to work with MongoDB from Python. If you
-need to access MongoDB in a non-blocking manner or from co-routines, we
-recommend that you use the :doc:`Motor </motor>` driver instead.
+You can use PyMongo, the official MongoDB driver for synchronous Python 
+applications, to interact with MongoDB. Download it using 
```

Comment:
How about the following to avoid too many commas (interrupts the flow):

> Welcome to the documentation site for PyMongo, the official MongoDB driver for synchronous Python applications. You can add the driver to your application to work with MongoDB in Python.  Download it using...

Reply:
I like this. It's clearer and incorporates "Welcome..." 

Changed for PyMongo, Motor and the Reactive Streams driver